.changeset/shiny-schools-search.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
I didn't drastically improve the JSDoc comments so I've opted for no explicit release here. The future PRs will be more substantial for each individual SDK